inc/options/common/top_nav_bar.php

```diff
@@ -0,0 +1,122 @@
+<?php
+namespace W3TC;
+
+if ( ! defined( 'W3TC' ) ) {
+	die();
+}
+
+$config = Dispatcher::config();
+
+$settings_menu_array = array(
+	Util_UI::admin_url( 'admin.php?page=w3tc_general' )        => array( 'text' => esc_attr__( 'General Settings', 'w3-total-cache' ) ),
```

Comment:
In all of the lines in this code block (3 variables), escape the URLs (with `Util_UI::admin_url()`) and target (with `esc_attr()`) in the `for` loop below.  Use `__()` to translate the text and later escape with `esc_html()` in the `for` loop.